.editorconfig

```diff
@@ -21,7 +21,7 @@ indent_size = 2
 trim_trailing_whitespace = false
 
 [*.json]
-insert_final_newline = false
+insert_final_newline = true
```

Comment:
```suggestion
```

@joemcgill Instead of setting it to `true`, could we remove it so it will use the global configuration, which is already set to `true`?